backend/migrations/versions/999999999999_manually_add_delegation_for_one_of_users.py

```diff
@@ -0,0 +1,70 @@
+"""Manually add delegation for one of the users
+
+Revision ID: 999999999999
+Revises: 5d3bf271b78d
+Create Date: 2024-07-17 16:48:12.133033
+
+"""
+import json
+from alembic import op
+
+revision = "999999999999"
```

Comment:
why this strange revision number?

Reply:
I'm just really bad at generating random numbers ;-)